x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_retention_policy.tsx

```diff
@@ -10,54 +10,81 @@ import React, { useEffect, useMemo, useState, type FC } from 'react';
 import { i18n } from '@kbn/i18n';
 
 import { EuiFormRow, EuiSelect, EuiSpacer, EuiSwitch } from '@elastic/eui';
-
-import { KBN_FIELD_TYPES } from '@kbn/field-types';
-
-import { useAppDependencies } from '../../../../app_dependencies';
+import { toMountPoint } from '@kbn/react-kibana-mount';
+import { useAppDependencies, useToastNotifications } from '../../../../app_dependencies';
+import { ToastNotificationText } from '../../../../components';
+import { isLatestTransform, isPivotTransform } from '../../../../../../common/types/transform';
+import { useGetTransformsPreview } from '../../../../hooks';
 
 import { EditTransformFlyoutFormTextInput } from './edit_transform_flyout_form_text_input';
 import { useEditTransformFlyout } from './use_edit_transform_flyout';
+import { getErrorMessage } from '../../../../../../common/utils/errors';
 
 export const EditTransformRetentionPolicy: FC = () => {
```

Comment:
Could we add the info text when the switch is disabled on the edit form, in the same way we do in the details step of the creation wizard?

<img width="299" alt="image" src="https://github.com/elastic/kibana/assets/7405507/7ca7fdfe-15eb-478d-802b-aa024210cdd1">